packages/status-react/src/styles/GlobalStyle.tsx

```diff
@@ -1,159 +1,172 @@
-import { createGlobalStyle } from 'styled-components'
+// todo: remove, 'cuase 2 libs 1 thing
+// import { createGlobalStyle } from 'styled-components'
+import styled from 'styled-components'
 
-export const GlobalStyle = createGlobalStyle`
-  * {
-    box-sizing: border-box;
-    -webkit-font-smoothing: antialiased;
+// xxx?: how was this generated; template for reseting (stylesheets; normalize css; for apps)
+// xxx?: how does this work in embedded context
+// note: reseting styles; only for its childs
+// note: selector, e.g. "> *" : { // css }
+// note: iframe; e.g. Intercom; separate context in browser; not accesible from "parent" page; not applies style
+// note: iframe; difficult; not responsive; check height and width of iframe; pitfalls; not allows consumers to update styles; theme props
+// export const GlobalStyle = createGlobalStyle`
+export const Wrapper = styled.div`
```

Comment:
note: just trying if `styled-components` handle the specificity/ordering, contrary to `stitches`